engine/src/main/java/org/terasology/world/block/family/FreeformBlockFamilyFactory.java

```diff
@@ -34,10 +35,13 @@ public BlockFamily createBlockFamily(BlockFamilyDefinition definition, BlockBuil
 
     @Override
     public BlockFamily createBlockFamily(BlockFamilyDefinition definition, BlockShape shape, BlockBuilderHelper blockBuilder) {
-        if (shape.isCollisionYawSymmetric()) {
-            return symmetric.createBlockFamily(definition, shape, blockBuilder);
-        } else {
-            return horizontal.createBlockFamily(definition, shape, blockBuilder);
+        switch( shape.getBlockShapePlacement().getType() ) { //Fixme others...
```

Comment:
The fixme really doesn't help here. 

Instead it is better to add an explict handling for all known cases and have no default section. That way bug finding tools can warn about it once  there is a new constant that does not get handled